### ISSUE TITLE
Echo-Net HMI Browser Compatibility

### ISSUE THREAD

**Issue by stephankokkas** [DataBytes-Organisation/Project-Echo#115]
The current implementation for the HMI has been tested and deployed for viewing through a Chrome browser. 

**Other browsers such as Safari are currently not supported.**

Current work on Project-Echo components are frozen for submission and review. Work will resume late June to mid July.

**Comment by dipanshgarg21** [comment]
Yes, Its not compatible with Mozilla firefox as I was being shown a black screen after I clicked on the welcome page of the project echo, instead of showing the map.
